product_docs/docs/pgd/5/quickstart/further_explore_failover.mdx

```diff
@@ -375,13 +375,17 @@ With kaolin fully back in service, you can leave everything as it is. There's no
 If you want, you can make kaolin leader again by running:
 
 ```shell
-pgd switchover --group-name dc1_subgroup --node-name kaolin
+pgd group dc1_subgroup set leader kaolin
 ```
 
 This command returns kaolin to write lead. The application's updates will follow, as the proxies track the write leader.
 
 ## Proxy failover
 
+!!!Note
+The command `pgd show-proxies` is deprecated as if PGD v5.7. It is still accessible as a legacy command.
```

Comment:
Looks like a typo in `as if PGD v5.7.`. Probably it should be `as of PGD v5.7.0`